Month1/Week2/Section1/Exercise1/Main.cpp

```diff
@@ -28,19 +28,19 @@ class FirstClass
 
 void testOne()
 {
-	FirstClass fc1;
-	FirstClass fc2;
-	FirstClass fc3;
-	FirstClass fc4;
-	FirstClass fc5;
+	FirstClass entityOne;
+	FirstClass entityTwo;
+	FirstClass entityThree;
+	FirstClass entityFour;
+	FirstClass entityFive;
 }
 
 
 void testTwo()
 {
 	for (int i = 0; i < 5; i++)
 	{
-		FirstClass f6;
+		FirstClass entitySix;
```

Comment:
All the first ones make sense since they are in the same scope. However, keep in mind that functions have their own scope. There isn't really a reason to call this entitySix for two reasons:
1. You are creating this 5 times. Do you have 5 `entity6`?
2. There are no other entities in this function. There is no concept of entities 1-5 in the current scope. A project can have literal millions of lines of code, you wouldn't want to check if somewhere in the code there is an `entity1` to see if you should name your entity `entity2`. In your context, this is the only entity. What do you think?